### PR TITLE
Feature/cica 195

### PR DESCRIPTION
**Pull request by armoj**

### CODE REVIEW

File: src/lib/resource/sections/rep-confirmation-method.js [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+                            }
+                        },
+                        required: ['q-rep-confirmation-method']
+                    }
+                }
+            },
+            errorMessage: {
+                required: {
+                    'q-rep-confirmation-method':
+                        'Select how you want to get your confirmation message'
```
Comment (Julieallan) [inline src/lib/resource/sections/rep-confirmation-method.js:154]: @armoj should this error mirror the question content? ie 'Select how we should tell you we've got your application'

File: src/lib/resource/sections/rep-confirmation-method.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+                    maxLength: 50,
+                    format: 'email',
+                    errorMessage: {
+                        maxLength: 'Email address must be 50 characters or less',
+                        format:
+                            'Enter an email address in the correct format, like name@example.com'
+                    },
+                    meta: {
+                        classifications: {
+                            theme: 'main-applicant-details'
```
Comment (Julieallan) [inline src/lib/resource/sections/rep-confirmation-method.js:54]: @armoj should this theme be 'rep-details'

File: src/lib/resource/sections/rep-confirmation-method.js [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+                    maxLength: 20,
+                    format: 'mobile-uk',
+                    errorMessage: {
+                        format:
+                            'Enter a UK mobile phone number, like 07700 900 982 or +44 7700 900 982',
+                        maxLength: 'Telephone number must be 20 characters or less'
+                    },
+                    meta: {
+                        classifications: {
+                            theme: 'main-applicant-details'
```
Comment (Julieallan) [inline src/lib/resource/sections/rep-confirmation-method.js:70]: @armoj should this be 'rep-details'

Comment (armoj) [reply src/lib/resource/sections/rep-confirmation-method.js:70]: fixed

File: src/lib/resource/sections/rep-confirmation-method.js (hunk 2 above)
Comment (armoj) [reply src/lib/resource/sections/rep-confirmation-method.js:54]: fixed

File: src/lib/resource/sections/rep-confirmation-method.js (hunk 1 above)
Comment (armoj) [reply src/lib/resource/sections/rep-confirmation-method.js:154]: fixed

**Review by Julieallan — APPROVED** [review]
' Theme ' is correct. Thanks

**Review by Julieallan — APPROVED** [review]

**Review by Julieallan — APPROVED** [review]

**Pull request merged**